core/src/main/java/org/bitcoinj/core/TransactionOutPoint.java

```diff
@@ -213,7 +213,7 @@ public RedeemData getConnectedRedeemData(KeyBag keyBag) throws ScriptException {
      * @return outpoint with removed connectedOutput
```

Comment:
I think the JavaDoc needs a change in the description of this semantics, too.

Reply:
Fixed and force-pushed.